cdap-ui/app/tracker/controllers/preview-ctrl.js

```diff
@@ -38,19 +39,22 @@ class TrackerPreviewController {
       scope: this.$scope
     };
 
-    let entityType;
     if (this.$state.params.entityType === 'streams') {
-      entityType = 'stream';
+      this.entityType = 'stream';
     } else if (this.$state.params.entityType === 'datasets') {
-      entityType = 'dataset';
+      this.entityType = 'dataset';
     }
 
-    let query = 'SELECT * FROM ' + entityType + '_' + this.$state.params.entityId + ' LIMIT 500';
+    let query = 'SELECT * FROM ' + this.entityType + '_' + this.$state.params.entityId + ' LIMIT 500';
```

Comment:
Tried exporting ExploreTablesStore  and calling window.CaskCommon.ExploreTablesStore.getState() but its returning empty array of tables.
Do I also need to import ActionCreater here and call fetchTables method to get the tables list or am I missing something here ?